lib/internal/webstreams/adapters.js

```diff
@@ -82,6 +86,28 @@ const { UV_EOF } = internalBinding('uv');
 
 const encoder = new TextEncoder();
 
+const ZLIB_FAILURES = new SafeSet([
+  ...ObjectEntries(internalBinding('constants').zlib)
+    .map(({ 0: code, 1: value }) => (value < 0 ? code : null)).filter(Boolean),
```

Comment:
This is extremely hard to understand and read. Would you mind adding some comments and improve the readability of this declaration?